### PR TITLE
replace travis icon with GitHub Actions

### PR DESCRIPTION
**Pull request by mashehu**
Before:
Screenshot 2020-03-31 14 54 48
After:
Screenshot 2020-03-31 14 54 34

Should I maybe make it a bit smaller?

### CODE REVIEW

**Comment by ewels** [comment]
Nice! I drew some vector GitHub action logos, if we want to stick with `.svg`: [GitHub Actions Logo.zip](https://github.com/nf-core/nf-co.re/files/4409150/GitHub.Actions.zip)

Agree that it looks a little big as it is now 👍

**Comment by mashehu** [comment]
In related things, how about moving the heading to the left? With the image on the right it always looks off center too me:
Screenshot 2020-03-31 15 01 41

**Comment by ewels** [comment]
Agreed, left-aligned headings look nicer 👍  Especially if including Docker 😅

**Comment by mashehu** [comment]
> Nice! I drew some vector GitHub action logos, if we want to stick with `.svg`: [GitHub Actions Logo.zip](https://github.com/nf-core/nf-co.re/files/4409150/GitHub.Actions.zip)
> 
> Agree that it looks a little big as it is now 👍

Great! I was surprised to not find an svg option on the web, but didn't want to put the effort int to do it myself 🙂 
FYI The font in the svg of the withtext version is broken/doesn't load for me.

**Comment by mashehu** [comment]
New icon size:
Screenshot 2020-03-31 15 12 08
Maybe a bit more padding on the left?

**Comment by ewels** [comment]
Sorry, I made this in a blip when I didn't have a copy of illustrator and was fighting with alternative software..

* Square version of the logo with no text: [github-actions-logo-notext.svg.zip](https://github.com/nf-core/nf-co.re/files/4409227/github-actions-logo-notext.svg.zip)
* New SVG version with broken text: 
[github-actions-logo-withtext.svg.zip](https://github.com/nf-core/nf-co.re/files/4409235/github-actions-logo-withtext.svg.zip)

**Comment by ewels** [comment]
I take it that you prefer it unbounded instead of in its little square box?

**Comment by mashehu** [comment]
Yes, the boxed version looks for me too much like a mobile app icon.

**Comment by ewels** [comment]
Fair enough 👍 Then yes, I think that the size is good now but it could do with a bit more padding / margin (can probably do for all images?).

**Comment by mashehu** [comment]
Screenshot 2020-03-31 15 24 53

File: public_html/index.php [hunk 1]
```diff
@@ -71,8 +71,8 @@
         <div class="col-md-4 mb-5">
           <div class="card">
             <div class="card-body">
-              <h5 class="card-title">Documentation</h5>
-              <img class="float-right ml-2" height="100px" src="assets/img/docs.svg" />
+              <h5 class="card-title text-left">Documentation</h5>
```
Comment (ewels) [inline public_html/index.php]: Better to edit the CSS I think (it's left by default): https://github.com/nf-core/nf-co.re/blob/0b1b8d3cdf89040aa37e57d21852bff34006261d/public_html/assets/css/nf-core.css#L501-L503

**Pull request merged**